server/server.go

```diff
@@ -34,7 +35,14 @@ const (
 	CommitmentModeKey = "commitment_mode"
 )
 
+type SvrConfig struct {
+	Host string
+	Port int
+	PPROF bool
+}
+
 type Server struct {
+	pprof 	   bool
```

Comment:
embed SvrConfig within the server to avoid putting pprof in many places

Reply:
I thought about this but felt it was unnecessary since the other cfg values are processed into embed struct fields